src/frontends/pytorch/src/op/quantile.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/floor.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/minimum.hpp"
+#include "utils.hpp"
+
```

Comment:
```suggestion

namespace ov {
namespace frontend {
namespace pytorch {
namespace op {

```
You forgot to open namespaces